.travis.yml

```diff
@@ -19,7 +19,7 @@ matrix:
 
 before_script:
 # If this command fails and you can't fix it, file an issue and add an exception to .nsprc
-- npm run nsp-check
+- if [[ "$TRAVIS_OS_NAME" != 'windows' ]]; then npm run nsp-check; fi
```

Comment:
This can be shortened to:

`[[ "$TRAVIS_OS_NAME" == windows ]] || npm run nsp-check`

(similarly below)

Reply:
Unfortunately travis doesn't like the shortcut syntax (my guess is that the `|` is being considered part of the yaml syntax), I'll land it as it is for now (we may double-check if there is a way to make the shortcut syntax to work in a travis yaml file).